pkg/generate/code/set_sdk.go

```diff
@@ -974,14 +994,16 @@ func setSDKForSlice(
 	//
 	//  f0elem.SetMyField(*f0iter)
 	containerFieldName := ""
+	sourceAttributePath := sourceFieldPath
 	if targetShape.MemberRef.Shape.Type == "structure" {
 		containerFieldName = targetFieldName
+		sourceAttributePath = sourceFieldPath+"."
```

Comment:
do we have a unit test that can stress this code path? I don't believe the unit test about User.Passwords will hit this.

Reply:
This path is unit tested in `TestSetSDK_MQ_Broker_Create` test for Users field.